posthog/temporal/data_imports/pipelines/sql_database/__init__.py

```diff
@@ -26,6 +26,70 @@
     SqlDatabaseTableConfiguration,
 )
 
+POSTGRES_TO_DLT_TYPES = {
+    # Text types
+    "char": "text",
+    "character": "text",
+    "varchar": "text",
+    "character varying": "text",
+    "text": "text",
+    "xml": "text",
+    "uuid": "text",
+    "cidr": "text",
+    "inet": "text",
+    "macaddr": "text",
+    "macaddr8": "text",
+    "tsvector": "text",
+    "tsquery": "text",
+    # Bigint types
+    "bigint": "bigint",
+    "bigserial": "bigint",
+    # Boolean type
+    "boolean": "bool",
+    # Timestamp types
+    "timestamp": "timestamp",
+    "timestamp with time zone": "timestamp",
+    "timestamp without time zone": "timestamp",
+    # Complex types (geometric types, ranges, json, etc.)
+    "point": "complex",
+    "line": "complex",
+    "lseg": "complex",
+    "box": "complex",
+    "path": "complex",
+    "polygon": "complex",
+    "circle": "complex",
+    "int4range": "complex",
+    "int8range": "complex",
+    "numrange": "complex",
+    "tsrange": "complex",
+    "tstzrange": "complex",
+    "daterange": "complex",
+    "json": "complex",
+    "jsonb": "complex",
+    # Double types
+    "real": "double",
+    "double precision": "double",
+    "numeric": "double",
+    "decimal": "double",
```

Comment:
Some of these (if not all?) should be `decimal` instead of `double` - https://dlthub.com/docs/general-usage/schema#data-types